### DISCUSSION TITLE
Powitanie!

### DISCUSSION THREAD

**Discussion by wi5nia** [AzureCommunityPL/group#14] [Ogólna]
Drodzy Grupowicze/Grupowiczki,

witajcie w nowym miejscu w sieci dla naszej społeczności 😃 mamy nadzieje, że będzie sie ono sprawowało dużo lepiej niż poprzednie - pamiętajcie, że póki co jest to wersja beta tej funkcjonalności, ale do odważnych świat należy, a my jako osoby zajmujace się nowoczesnymi technologiami takich rzeczy się nie boimy!

Kilka spraw organizacyjnych:

- w sekcji [Code](https://github.com/AzureCommunityPL/group) znajdziecie podstawowe [Readme](https://github.com/AzureCommunityPL/group/blob/master/README.md), [Code of Conduct](https://github.com/AzureCommunityPL/group/blob/master/CODE_OF_CONDUCT.md) czy też katalog [faq ](https://github.com/AzureCommunityPL/group/tree/master/faq) gdzie możecie znaleźć niezbędne informacje o tym jak funkcjonować w nowym miejscu
- bardzo zachęcamy do dania gwiazdki naszemu repo
![image](https://user-images.githubusercontent.com/12593852/90387115-e2e2ad80-e085-11ea-8e60-a64f199473f1.png)
- jeśli macie jakies pytania, sugestie co do nowego miejsca śmiało pytajcie pod tym postem 👍

**Comment by senionit** [comment]
Cześć,

Na początek dzięki za migrację na platformę developerską.

a teraz kilka uwag z mojej strony:
 * discussions nie jest widoczne w aplikacji mobilnej github, a jak się otwiera githuba np. na ipadzie to z automatu otwiera się aplikacja. Warto dodać tę informację do faq. Pomaga przytrzymanie linku i wybranie z menu safari otwórz w nowej karcie.
* faq jest zestawem pytań i odpowiedzi odnośnie azure, chciałbym żeby na początku faq były informacje o samej grupie, jak się zapisać, jak uczestniczyć i które elementy grupy odbywają się na jakiej platformie.

**Reply by wi5nia** [reply]
1. No jest to funkcjonalnosc poki co "beta" wiec GH pracuje nad kolejnymi rzeczami i zapewne bedzie w aplikacji. W [FAQ](https://github.com/AzureCommunityPL/group/tree/master/faq) masz napisane jak dodac zmiane - mozesz sam to zrobic i kontrybuowac do community 😃 
2. Mamy to w planach. Znowu wracajc do powyzszego - jest to beta discussions i czekamy na kilka funkcjonalnosci, ktore ulatwia nam propagowanie takich informacji

**Reply by kaluzaaa** [reply]
@senionit co do 2. to tutaj zaczynają się dziać rzeczy, więc zapraszamy :) https://github.com/AzureCommunityPL/azurecommunitypl.github.io/issues

**Comment by wi5nia** [comment]
Dla osób, które nie były na spotkaniu lub też nie zauważyły - uruchomiliśmy ponownie możliwość pisania postów na grupie na FB - [Microsoft Azure User Group Poland](https://www.facebook.com/groups/azureugpl), ale tutaj także można prowadzić dalej rozmowy :)
